_pages/meetings.md

```diff
@@ -2,7 +2,7 @@
 layout: page
 title: Meetings
 permalink: /meetings/
-order: 2
+order: 3
```

Comment:
This is a typo fix, not required for pagination. Both "meetings" and "docs" used to have `order: 2`, but "meetings" appears after "docs" in the nav bar so this is the right disambiguation.